README.md

```diff
@@ -1,2 +1,19 @@
 # hw01
 Repository for homework 1
+
+# "Biography of Jimin Yoon"
+I'm a first year doctoral student in **sociology** department. My work explores the relationship between state and society, war and capitalism, and democracy and authoritarianism. My current research concerns the institutionalizing process of labor movement in the U.S. 
+
+*Research Interests*:
+* Comparative/Historical/Macro Sociology
+* Culture
+* Political Sociology
+* Methodology
+* Social Change
+* Social Theory
+
+*My Dog*
+![Mango](https://scontent.ford1-1.fna.fbcdn.net/v/t31.0-8/883544_142567279243864_1444921524_o.jpg?_nc_cat=0&oh=7759dbc1c22819ff2aff99b1a8e00e3e&oe=5B68B3A2)
+
+*Email* 
+[jiminyoon@uchicago.edu](mailto:jiminyoon@uchicago.edu)
```

Comment:
Great work with the email forwarding link!

Reply:
Evaluation HW-01 for Jimin Yoon 
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X     |              |            |
| **Coding strategy**         |     X     |              |            |
| **Presentation**            |     X     |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X     |              |            |

Remarks:

* Nice job on the first homework! Clear README with multiple markdown elements, especially impressed with your email link.
* Instead of using a long URL link for your picture, can save an image of Mango in your repo and call it from there as well.
* Always good to change your commit information so you know what changes your making with each.
* Your dog is absurdly cute! :)